appservice/src/SiteWrapper.ts

```diff
@@ -362,12 +381,76 @@ export class SiteWrapper {
         await this.waitForDeploymentToComplete(kuduClient, outputChannel);
     }
 
+    private async getJsonRequest(url: string, requestOptions: WebResource, node: IAzureNode): Promise<Object[]> {
+        // Reference for GitHub REST routes
+        // https://developer.github.com/v3/
+        // Note: blank after user implies look up authorized user
+        try {
+            const gitHubResponse: string = await requestP.get(url, requestOptions);
+            return JSON.parse(gitHubResponse);
+        } catch (error) {
+            if (error.message.indexOf('401 - "{\"message\":\"Bad credentials\",\"documentation_url\":\"https://developer.github.com/v3\"}"') > -1) {
```

Comment:
Use `parseError(error).message`. It's safer and handles the case where message might be undefined.

Also do we need to look for that _entire_ message or can you just check for 401 or "Bad credentials"? I'm worried they might slightly change the message and then this check no longer works

Reply:
I'll use "Bad credentials".  Seems like a safe bet to me.